packages/skin/src/routes/_index/component/lightbox-dialog/+page.marko

```diff
@@ -1,504 +1,1077 @@
                     <div class="lightbox-dialog__main">
-                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-                            magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-                            consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-                            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-                        <p><a href="https://www.ebay.com">www.ebay.com</a></p>
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                        </p>
+                        <p>
+                            <a href="https://www.ebay.com">
+                                www.ebay.com
+                            </a>
+                        </p>
                     </div>
                 </div>
             </div>
         </div>
     </div>
 
-    <highlight-code type="html" >
-<div aria-labelledby="lightbox-dialog-title" aria-modal="true" class="lightbox-dialog lightbox-dialog--expressive" hidden role="dialog">
-    <div class="lightbox-dialog__window">
-        <div class="lightbox-dialog__image" style="background-image:url(src/routes/static/img/tb-landscape-pic.jpg)"></div>
-        <div class="lightbox-dialog__header">
-            <h2 id="lightbox-dialog-title" class="large-text-1 bold-text">Heading</h2>
-            <button aria-label="Close dialog" class="icon-btn lightbox-dialog__close" type="button">
-                <svg aria-hidden="true" class="icon icon--16" height="16" width="16">
-                    <use href="#icon-close-16"></use>
-                </svg>
-            </button>
-        </div>
-        <div class="lightbox-dialog__main">
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-                magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-                consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-                Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-            <p><a href="https://www.ebay.com">www.ebay.com</a></p>
+    <highlight-code type="html">
```

Comment:
Are there any substantive differences in these blocks or is this merely formatting adjustments?

Reply:
I ran it through prettier
<img width="530" alt="Screenshot 2025-05-15 at 10 25 45 AM" src="https://github.com/user-attachments/assets/f3a99f83-e099-4168-bc4b-e9783138c6af" />
You can toggle whitespace changes. There shouldn't be any changes really other than class names

Reply:
The main changes should be that `dialog__window--fade` is no longer used, and we use use `dialog__window--animate` by default. 
Also `dialog__mask--fade-slow` has been removed from examples (but still in code, deprecated)